### ISSUE TITLE
Entry details pane too dark

### ISSUE THREAD

**Issue by smcyr** [craftcms/cms#2830]
### Description
I feel that the details pane (#main-container #main #main-content #details) has a background color too dark (#d5d8dd) and it make it seems like it's "disabled". And the contrast for the background color and the text color seems a bit poor (https://webaim.org/resources/contrastchecker/). So, in my opinion, the text should be darker and the background lighter.
Thanks!

### Additional info
- Craft version: 3.0.5
- PHP version: 7.2

**Comment by carlcs** [comment]
This can be closed now that the colors have been updated?

**Comment by brandonkelly** [comment]
Yep :)
